user/pages/03.commerce2/02.developer-guide/12.recurring/01.subscriptions-overview/docs.md

```diff
@@ -0,0 +1,69 @@
+* Billing schedules, the billing schedule plugin, Billing periods, and the Prorater plugin
+* The Subscription type plugin and Charges
+* Recurring order types and order item types
+* The PurchasableEntitySubscriptionTrait and the Initial order
+
+## Billing Schedules
+Each subscription has a billing schedule. Billing schedules are configuration entities that are created through the admin UI: `Commerce >> Configuration >> Payment`. Each billing schedule has a type, which is either prepaid or postpaid (charge at the beginning or at the end of the billing period). 
+
+### Billing Schedule plugin
+A billing schedule also has a billing schedule plugin. Two billing schedule plugins are included in the Commerce Recurring module: Rolling and Fixed. Both rolling and fixed billing schedules have an Interval (some number of hours, days, weeks, months, or years). Fixed billing schedules with monthly intervals also have a Start day (1 - 31). Fixed billing schedules with yearly intervals also have a Start month (Jaunuary - December). 
+
+Suppose a new annual subscription is purchased on October 12th. If the first billing period should be October 12 - October 12, then you should set up a "Rolling" billing schedule. All subsequent billing periods for this subscription will also begin and end on October 12th. Suppose, on the other hand, that you want the annual subscription to start on January 1st every year. Then you should set up a "Fixed" billing schedule; the first billing period will run from October 12 to January 1st, and subsequent billing periods will start and end on January 1st. The billing schedule plugin is configured through the admin UI:
+
+![Fixed_annual_billing_schedule](../images/Fixed_annual_billing_schedule.png)
+
+The primary function of a billing schedule plugin is to generate billing periods:
+`generateFirstBillingPeriod()` generates the first billing period for a subscription, given a start date.
+`generateNextBillingPeriod()` generates subsequent billing periods for a subscription, given a start date and the current billing period.
+
+### What is a billing period?
+A billing period is an object that represents a span of time. Examples:
+ * - Oct 14th 14:56:20 - Nov 14th 14:56:20
+ * - May 1st 00:00:00 - June 1st 00:00:00
+ * - June 1st 00:00:00 - July 1st 00:00:00
+
+Billing periods are contiguous and represent [half-open ranges](http://wrschneider.github.io/2014/01/07/time-intervals-and-other-ranges-should.html) (the end date is not included in the duration).
+
+#### Billing period field type
+The Commerce Recurring module defines a custom "Billing Period" field type that is used by "recurring" type orders and "recurring" type order items. The field has two properties, the Start date and the End date, both of which are timestamps. A custom field widget and formatter are also provided for the billing period field type.
```

Comment:
We should not be explaining the field type until we actually describe the data model of recurring orders.